### PR TITLE
New Testing for the backend

### PR DESCRIPTION
**Pull request by Michiocre**

### CODE REVIEW

**Comment by Michiocre** [comment]
Change in Dashboard is because when developing on a different port then 80 it wouldnt work

File: backend/app.js [hunk 1]
```diff
@@ -197,6 +199,7 @@ io.on('connection', (socket) => {
 	* @param {string} roomId Id of the room
 	*/
     socket.on('roomId', data => {
+        console.log(data)
```
Comment (Gabsii) [inline backend/app.js]: rm console.log

File: frontend/src/pages/Dashboard.jsx [hunk 2]
```diff
@@ -47,7 +47,7 @@ class Dashboard extends Component {
         }
 
         //Gets rid of the search in window.location
-        var myNewURL = window.location.protocol + '//' + window.location.hostname + window.location.pathname;
+        var myNewURL = window.location.protocol + '//' + window.location.hostname + ':' + window.location.port + window.location.pathname;
```
Comment (Gabsii) [inline frontend/src/pages/Dashboard.jsx]: was this tested with 80 as a port? how does the url look like? will the port be sent? e.g. `spoti-vote.com:80/dashboard` ?

Comment (Michiocre) [reply frontend/src/pages/Dashboard.jsx]: no idee my port 80 was taken :(

Comment (Gabsii) [reply frontend/src/pages/Dashboard.jsx]: nice

**Pull request merged**